src/client.js

```diff
@@ -60,3 +61,28 @@ function refresh_token (_refresh_token) {
             return result;
         });
 }
+
+function revoke_token (token, client_id, type = '') {
+    const client_key = cache.find_key('client', {token});
+    const server_key = cache.find_key('server', {token});
+    const payload = {
```

Comment:
Can be re-written as
```suggestion
    const payload = {
    	client_id,
    	type
    };
```

Reply:
Wow. :scream:  Anyway, I already updated this one. Kindly review it again. Thaaaankk yooou. :smile:

Reply:
I think you missed to update the `client_id`? :sweat_smile:

Reply:
Ahh. Right. Apologies. Will include this on my next commit. :sweat_smile: